modules/claims_api/spec/requests/v2/veterans/rswag_disability_compensation_request_spec.rb

```diff
@@ -56,7 +56,7 @@
       parameter SwaggerSharedComponents::V2.body_examples[:disability_compensation]
 
       describe 'Getting a successful response' do
-        response '200', 'Successful response with disability' do
+        response '202', 'Successful response with disability' do
           schema JSON.parse(Rails.root.join('spec', 'support', 'schemas', 'claims_api', 'v2', 'forms',
```

Comment:
With the update to using 526.json this was the spot where somethng like this will allow them tomatch up since the response is nested in data: but the schema file is not

```
form_data = JSON.parse(Rails.root.join('modules', 'claims_api', 'config', 'schemas', 'v2', '526.json').read)
          nested_form_data =  {data: form_data}

          schema nested_form_data
```